.github/workflows/turborepo-library-release.yml

```diff
@@ -0,0 +1,113 @@
+name: Turborepo Library Release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    defaults:
+      run:
+        shell: bash -leo pipefail {0}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        settings:
+          - host: macos-latest
+            target: "aarch64-apple-darwin"
+          - host: macos-latest
+            target: "x86_64-apple-darwin"
+          - host: ubuntu-latest
+            target: "aarch64-unknown-linux-gnu"
+            setup: |
+              sudo apt update
+              sudo apt install -y g++-aarch64-linux-gnu libc6-dev-arm64-cross
+              echo "CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER=aarch64-linux-gnu-gcc" >> $GITHUB_ENV
+              echo "CC_aarch64_unknown_linux_gnu=aarch64-linux-gnu-gcc" >> $GITHUB_ENV
+              echo "CXX_aarch64_unknown_linux_gnu=aarch64-linux-gnu-g++" >> $GITHUB_ENV
+          - host: ubuntu-latest
+            target: "x86_64-unknown-linux-gnu"
+          - host: windows-latest
+            target: "aarch64-pc-windows-msvc"
+          - host: windows-latest
+            target: "x86_64-pc-windows-msvc"
+
+    runs-on: ${{ matrix.settings.host }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Setup Rust
+        uses: ./.github/actions/setup-rust
+        with:
+          targets: ${{ matrix.settings.target }}
+
+      - name: Setup Node
+        uses: ./.github/actions/setup-node
+        with:
+          enable-corepack: false
+
+      - name: Setup toolchain
+        run: ${{ matrix.settings.setup }}
+        if: ${{ matrix.settings.setup }}
+
+      - name: Build native library
+        run: |
+          cd packages/turbo-repository
+          mkdir -p js/dist
```

Comment:
shouldn't the build script create the build dir?

Reply:
Yeah, the build script does create this folder now. I'll delete this line from the GH workflow.